source/agora/consensus/EnrollmentManager.d

```diff
@@ -929,6 +951,28 @@ public class EnrollmentManager
     }
 }
 
+/// This is a cycle seed commonly used for testing.
+version (unittest)
+{
+    static immutable CommonCycleSeed = Hash(`0xb7f3802d774665c6ccb4d24f3e6128542185f847e6da4e9c5a0254cfeb885d2536a89be8b882f0917f3ed43dae4760bb3a85b4c5a9e1871872a794fcf5e5236d`);
+}
+
+/// Each `PreImageCycle` for ramdon keys can be created just with a cycle seed without generating
+/// 5 million preimages. These pre-defined cycle seeds are used for that.
+version (unittest)
+{
+    static immutable Hash[] NodeCycleSeeds = [
+        Hash(`0x3a481cb1576d79002755239b8d4019587d7e5394ddd448f92d5dca74baf742f4899dd53daefda9136e9d3cc6887b3456ffcfc85cfc711e5cf8659998047771cb`),
```

Comment:
Logically, I think those would belong to `genesis/Test.d`. But I am still reviewing, any reason why they are in this module ?

Reply:
The values are kind of random just to be used in the tests in the `EnrollmentManager` module.

Reply:
I think you could put the values in https://github.com/bosagora/agora/pull/2291#discussion_r673655771 in `genesis/Test.d` and use that everywhere, right ?

Reply:
I think so. It may not be needed. I'll look into it.

Reply:
Updated.

Reply:
I moved it to PreImage.d. I tried to move it to `Base.d` but it gets the cycling dependency.